docker-compose.yml

```diff
@@ -90,6 +92,40 @@ services:
       - 3000:3000
     restart: unless-stopped
 
+  lnd:
+    container_name: lnd
+    image: lightninglabs/lnd:v0.15.5-beta
+    depends_on:
+      - bitcoin
+    command:
+      - --configfile=/var/lib/lnd/lnd.conf
+    volumes:
+      - ./faucet/lnd.conf:/var/lib/lnd/lnd.conf
+      - ./faucet/macaroons.db:/root/.lnd/data/chain/bitcoin/regtest/macaroons.db
+      - bitcoin:/data
+    environment:
+      HOME: /data
+    restart: unless-stopped
+    stop_grace_period: 5m30s
+    ports:
+      - "9735:9735" # p2p
+      - "10009:10009" # grpc"
+      - "18080:18080" # rest"
+
+  faucet:
+    image: nginx
+    container_name: faucet
+    ports:
+      - 8080:80
+    volumes:
+      - ./faucet/faucet.html:/usr/share/nginx/html/faucet/index.html
+      - ./faucet/open_channel.html:/usr/share/nginx/html/channel/index.html
+      - ./faucet/default.conf:/etc/nginx/conf.d/default.conf
```

Comment:
I guess my two comments above are obsolete because we still need these files :D 🙈

Reply:
yes, unfortunately.. alternatively we could build a small docker image, that contains these files.